### DISCUSSION TITLE
Where do I find the shop_id?

### DISCUSSION THREAD

**Discussion by sebpowell** [etsy/open-api#1166] [Q&A] [answered]
Hello, 

Just getting started with the Etsy API – a number of endpoints reference a `shop_id` param, but nowhere as far as I can see, does it specify where to find this value. 

Any help would be greatly appreciated, thank you in advance.

**Comment by Gareth064** [comment, marked as answer]
Once you've done the Auth dance and got an Auth Token. The first part of the token will be the "User_ID" of the shop owner.

Once you get that, you can call https://developers.etsy.com/documentation/reference/#operation/getShopByOwnerUserId using the User_ID, and that will get you the ShopId.

**Reply by sebpowell** [reply]
@Gareth064 ah, I missed that in the docs – that makes sense, thank you!
